scripts/admin/contributors.py

```diff
@@ -183,9 +215,20 @@ def collect_contributors(repository, start_date, end_date):
     except:
         print("Ensure that the end date is in YYYY-MM-DD format.")
         sys.exit(1)
+    print('BEGIN DATA COLLECTION... (this can take some time)')
     tic = perf_counter()
-    contrib_info = collect_contributors(repository, start_date, end_date)
+    contrib_info, author_name_map, tags_only = collect_contributors(
+        repository, start_date, end_date
+    )
     toc = perf_counter()
     print(f"\nCOLLECTION COMPLETE. Time to completion: {toc - tic:0.4f} seconds")
     print(f"\nContributors between {sys.argv[2]} and {sys.argv[3]}:")
-    pprint.pprint(contrib_info)
+    for item in author_name_map.values():
+        print(item)
+    print("\nOnly GitHub handles are available for the following contributors:")
+    for tag in tags_only:
+        print(tag)
+    json_filename = f"contributors-{sys.argv[2]}-{sys.argv[3]}.json"
```

Comment:
Two questions: 
- should this file include the repo name as well?
- should you check that the file does not exist before (blindly) overwriting it?

Reply:
Addressed. (reluctantly)